ts/parts/Series.ts

```diff
@@ -76,6 +76,7 @@ declare global {
             public linkedSeries: Array<Series>;
             public markerGroup?: SVGElement;
             public name: string;
+            public opacity?: number;
```

Comment:
Why do we need this series property? Can't we just read it directly from the options where we use it?

Reply:
See my explanation below.